README.md

```diff
@@ -46,10 +57,18 @@ In a second terminal, start the node of Alice with
 ```sh
 ./perun-polkadot-demo demo --config alice.yaml
 ```
-and in a third terminal, start the node of Bob with
+or in Docker:
+```sh
+docker run --rm -it -v $(pwd):/cfg --net=host perun-polkadot-node demo --config /cfg/alice.yaml --network /cfg/network.yaml
+```
+In a third terminal, start the node of Bob with
 ```sh
 ./perun-polkadot-demo demo --config bob.yaml
 ```
+or in Docker:
+```sh
+docker run --rm -it -v $(pwd):/cfg --net=host perun-polkadot-node demo --config /cfg/bob.yaml --network /cfg/network.yaml
+```
```

Comment:
`--net=host` does not work well across different OSes. Basically only works on linux, see [here](https://docs.docker.com/network/host/). Use `-p` with the appropriate port instead.

Could you move the docker instructions together below the local execution instructions? That is, keep the test as before. Only insert after the local instructions for Bob (make sure to replace --net=host with -p and the appropriate port):

Alternatively you can start the clients via docker.
```sh
docker run --rm -it -v $(pwd):/cfg --net=host perun-polkadot-node demo --config /cfg/alice.yaml --network /cfg/network.yaml // In one terminal window.
docker run --rm -it -v $(pwd):/cfg --net=host perun-polkadot-node demo --config /cfg/bob.yaml --network /cfg/network.yaml // In a second terminal window.
```

Reply:
I like the restructuring, but for the `-p` thing to work out, the user would have to modify the `networking.yaml`.  
Currently *Alice* and *Bob* both run on *localhost*, but if we use `-p` then the user would have replace the IPs in the `networking.yaml` with `172.17.0.2` and `172.17.0.3` which makes it harder to use.

Reply:
I think it should work for `localhost` or `127.0.0.1`?

Reply:
I added a note that it will not work on all systems.